traceabilityViewer/traceabilityViewer/settings.py

```diff
@@ -0,0 +1,136 @@
+"""
+Django settings for traceabilityViewer project.
+
+Generated by 'django-admin startproject' using Django 4.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+from neomodel import config
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-9@1+y@2v04x9t4r=ih5-%j7ueh(84j6($3tt6js&pd%ozl91du"
```

Comment:
What's this for?

Reply:
Django will refuse to start if `SECRET_KEY` is not set.
https://docs.djangoproject.com/en/5.0/ref/settings/#std:setting-SECRET_KEY

Reply:
it is needed. Should end up in .env file and here you should use `python-decouple` with `config` call.

Reply:
Should I also use `python-decouple` for the other environment variables?
I use for the other variables `os.environ.get("VARIABLE")`

Reply:
Yes. Usually, `python-decouple` is used so that you can easily switch among environments and it on the end does the same call if it cannot find the variable in the configuration file.